xonsh/pyghooks.py

```diff
@@ -177,7 +177,7 @@ def color_name_to_pygments_code(name, styles):
             fgcolor = color
         else:
             fgcolor = styles[getattr(Color, color)]
-        res = "bg:" + fgcolor
+        res = "bg:" + fgcolor if fgcolor != "noinherit" else "noinherit"
```

Comment:
could you add a bracket around bg and fgcolor ? to make it clear the precedence

Reply:
Added in a more verbose conditional to make it explicit